types/block_test.go

```diff
@@ -65,6 +66,7 @@ func makeCommit(blockID BlockID, height int64, round int,
 		Round:            round,
 		Type:             VoteTypePrecommit,
 		BlockID:          blockID,
+		Timestamp:        time.Now().UTC(),
```

Comment:
Minor nit: could we make this `time.Now().Round(time.Second).UTC()`? lest we'll have some nanoseconds in there.

Reply:
It's all good. We want some... We round to milliseconds when hashing/signing it.

Reply:
I guess even digital computers are made of analog stuff, and there's a little fudge everywhere.  This is digital fudge?